:pencil: philosophy documentation (#13)
FBerthelot/component-test-utils: website/docs/concept.md

title: Documentation
---

Component-test-utils is an opinionated test framework.

## Why should you use it ?

It mocks your favorite framework in a way that allows to write sustainable and functional oriented tests of your components.

> As component-test-utils consider component as a black box, you won't be able to directly access to the state of your components.

Reviewer: ```suggestion
Because component-test-utils consider each component as a black box, you won't be able to get access to your components state.
```
